packages/core/backend/sirius-components-collaborative/src/main/resources/schema/omnibox.graphqls

```diff
@@ -0,0 +1,25 @@
+extend type Viewer {
+    omniboxCommands(contextEntries: [OmniboxContextEntry!]!, query: String!): OmniboxCommandsConnection!
+}
+input OmniboxContextEntry {
+    id: ID!
+    label: String!
+    kind: String!
+}
+
+type OmniboxCommandsConnection {
+    edges: [OmniboxCommandsEdge!]!
+    pageInfo: PageInfo!
+}
+
+type OmniboxCommandsEdge {
+    node: OmniboxCommand!
+}
+
+type OmniboxCommand {
+    id: ID!
+    label: String!
+    kind: String!
+    iconURL: [String!]!
```

Comment:
`iconURLs`

Reply:
I used `iconURL` because it is used everywhere, even when the API accepts many values.
I will change to `iconURLs` but we should update that everywhere it is needed later.

Reply:
Done